### PR TITLE
taskell: init at 1.3.2

### PR DESCRIPTION
**Pull request by matthiasbeyer**
@Profpatsch please ACK this, as you're in the git trailers.

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/applications/misc/taskell/default.nix [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  src = fetchFromGitHub {
+    owner = "smallhadroncollider";
+    repo = "taskell";
+    rev = version;
+    inherit sha256;
+  };
+
+  postPatch = ''${haskellPackages.hpack}/bin/hpack'';
+
+  # basically justStaticExecutables; TODO: use justStaticExecutables
```
Comment (Profpatsch) [inline pkgs/applications/misc/taskell/default.nix]: Dunno what we should do about this. I tried for about 5 minutes to get it to run with `haskellPackages.callPackage`, but just copying the arguments was easier. Would be nicer if we could use `justStaticExecutables`.

**Review by Profpatsch — COMMENTED** [review]
Okay from my side, I wrote the code after all. :)

**Comment by matthiasbeyer** [comment]
Status?

**Comment by Profpatsch** [comment]
lgtm

**Pull request merged**